src/manifests_operator.py

```diff
@@ -8,11 +8,24 @@
 from hashlib import md5
 from typing import Dict
 
-from ops.manifests import ManifestLabel, Manifests
+from lightkube.codecs import AnyResource
+from ops.manifests import ManifestLabel, Manifests, Patch
 
 log = logging.getLogger()
 
 
+class PatchPluginName(Patch):
```

Comment:
My thought is that this would be more automated so that we don't need to update the YAML files from upstream, i.e. we can just copy them as-is from Intel's repo, and then limit any changes/customization to the charm code.

We could also reuse this for other device types once we copy those in. They all use the same `-sample` in their naming. See: https://github.com/intel/intel-device-plugins-for-kubernetes/tree/release-0.29/deployments/operator/samples